src/plugins/intel_gna/tests/functional/shared_tests_instances/single_layer_tests/convolution.cpp

```diff
@@ -13,21 +13,115 @@ using namespace LayerTestsDefinitions;
+    result << "netPRC=" << netPrecision.name() << "_";
+    result << "inPRC=" << inPrc.name() << "_";
+    result << "outPRC=" << outPrc.name() << "_";
+    result << "inL=" << inLayout << "_";
+    result << "outL=" << outLayout << "_";
+    result << "trgDev=" << targetDevice;
+    for (auto const& config_entry : config) {
+        result << "_config_entry=" << config_entry.first << "_" << config_entry.second;
     }
-};
+    return result.str();
+}
 
-TEST_P(gna_convolution_layerTest, CompareWithRefs) {
-    Run();
+void Convolution2DLayerTest::SetUp() {
+    convSpecificParams convParams;
+    std::vector<size_t> inputShape;
+    auto netPrecision = InferenceEngine::Precision::UNSPECIFIED;
+    std::tie(convParams, netPrecision, inPrc, outPrc, inLayout, outLayout, inputShape, targetDevice, configuration) =
+        this->GetParam();
+    ngraph::op::PadType padType;
+    InferenceEngine::SizeVector kernel, stride, dilation;
+    std::vector<ptrdiff_t> padBegin, padEnd;
+    size_t convOutChannels;
+    std::tie(kernel, stride, padBegin, padEnd, dilation, convOutChannels, padType) = convParams;
+    auto ngPrc = FuncTestUtils::PrecisionUtils::convertIE2nGraphPrc(netPrecision);
+    auto params = ngraph::builder::makeParams(ngPrc, {inputShape});
+    auto paramOuts =
+        ngraph::helpers::convert2OutputVector(ngraph::helpers::castOps2Nodes<ngraph::op::Parameter>(params));
+    std::vector<float> filter_weights;
+
+    auto filter_size = std::accumulate(std::begin(kernel), std::end(kernel), 1, std::multiplies<size_t>());
+    filter_weights =
+        CommonTestUtils::generate_float_numbers(convOutChannels * inputShape[1] * filter_size, -0.5f, 0.5f);
+
+    auto conv =
+        std::dynamic_pointer_cast<ngraph::opset1::Convolution>(ngraph::builder::makeConvolution(paramOuts[0],
+                                                                                                ngPrc,
+                                                                                                kernel,
+                                                                                                stride,
+                                                                                                padBegin,
+                                                                                                padEnd,
+                                                                                                dilation,
+                                                                                                padType,
+                                                                                                convOutChannels,
+                                                                                                false,
+                                                                                                filter_weights));
+    ngraph::ResultVector results{std::make_shared<ngraph::opset1::Result>(conv)};
```

Comment:
Could you please use the ov:: namespace, ngraph is legacy one, and it is better to use the latest opset in the tests:
using namespace ov::opset10;
....
ov::ResultVector results{std::make_shared<Result>(conv)};